test/Recipe-test.js

```diff
@@ -125,7 +57,71 @@ describe('Recipe Class', function() {
       ]
     };
 
-    recipe = new Recipe(recipeDatum);
+    recipe = new Recipe(recipeDetails);
+  });
+
+  it('should have be a function', function() {
+    expect(Recipe).to.be.a('function');
   });
 
+  it('should be an instance of Recipe', function() {
+    expect(recipe).to.be.an.instanceOf(Recipe);
+  });
+
+  desbribe('Recipe constructor properties', function() {
+    it('should have an id', function() {
+      expect(recipe.id).to.equal(595736);
+    });
+
+    it('should store an image url', function() {
+      expect(recipe.image).to.equal("https://spoonacular.com/recipeImages/595736-556x370.jpg");
+    });
+
+    it('should be able to all ingredient\'s information', function() {
+      const ingredients = recipe.getIngredients(recipeDetails.ingredients);
+
+      expect(recipe.ingredients).to.be.an('array');
+      expect(recipe.ingredients.length).to.deep.equal(3);
+      expect(ingredients).to.deep.equal(recipe.ingredients);
+      expect(recipe.ingredients[0]).to.be.an.intanceOf(Ingredient);
+      expect(recipe.ingredients[0].name).to.equal('wheat flour');
+    });
+
+    it('should store a list of instructions', function() {
+      expect(recipe.instructions).to.be.an('array');
+      expect(recipe.instructions[0]).to.be.an('object');
+      expect(recipe.instructions).to.deep.equal(recipeDetails.instructions);
```

Comment:
Are the tests on lines 91 and 92 needed when line 93 is already testing for those things? I understand that adding the extra tests may help pinpoint where an error is occurring, however if recipeDetails.instructions isn't being assigned properly by the recipe constructor, then it doesn't seem like the first two tests will help get that done. They will always fail if the constructor fails to pass in recipeDetails.instructions, and they will always pass if the constructor succeeds in passing through the array. I can't think of any case (off the top of my head) of them failing even though the constructor succeeded or vice verse.